Assets/Scripts/PlayerController/PlayerMovement.cs

```diff
@@ -145,6 +155,16 @@ public bool GetJumpedInCurrentFrame()
         return m_JumpedInCurrentFrame;
     }
 
+    public void SetUnmovable(bool boolean)
+    {
+        m_cannotMove = boolean;
+    }
+
+    public bool CheckIfUnmovable()
```

Comment:
Make sure getters are appended with `Get`. `GetIsMovementLocked` or `GetMovementLocked`. This makes it easy to filter by available getters in IntelliSense. Alternatively, convert to a C# property.